Fix:

Undefined symbols for architecture x86_64:
  "_WebPGetFeaturesInternal", referenced from:
      _webp_loader_new_from_mapping in chafa-webp-loader.o
  "_WebPGetInfo", referenced from:
      _webp_loader_new_from_mapping in chafa-webp-loader.o
ld: symbol(s) not found for architecture x86_64

https://github.com/hpjansson/chafa/issues/152
https://github.com/hpjansson/chafa/commit/1a22c4e2c3acb9d5fea5b6a7a74d0bf3fc9102a7
--- configure.ac.orig
+++ configure.ac
@@ -150,7 +150,7 @@ AS_IF([test "$with_tools" != no], [
     ,
     with_webp=yes)
   AS_IF([test "$with_webp" != no], [
-    PKG_CHECK_MODULES(WEBP, [libwebpdemux],,
+    PKG_CHECK_MODULES(WEBP, [libwebp libwebpdemux],,
       missing_rpms="$missing_rpms libwebp-devel"
       missing_debs="$missing_debs libwebp-dev"
       with_webp=no)])
